cheat.php

```diff
@@ -186,11 +186,15 @@
 		$Time = ( $Data[ 'next_level_score' ] - $Data[ 'new_score' ] ) / GetScoreForZone( [ 'difficulty' => $Zone[ 'difficulty' ] ] ) * ( $WaitTime / 60 );
 		$Hours = floor( $Time / 60 );
 		$Minutes = $Time % 60;
+		$Date = date_create();
+		
+		date_add($Date,date_interval_create_from_date_string($Hours . " hours"));
+		date_add($Date,date_interval_create_from_date_string($Minutes . " minutes"));
```

Comment:
You could do `$Hours . " hours + " . $Minutes . " minutes"` to parse only once: http://php.net/manual/en/dateinterval.createfromdatestring.php

Reply:
Implementing... Good catch